src/components/shared/SearchInput.js

```diff
@@ -115,7 +119,7 @@ class SearchInput extends Component {
         placeholder,
         value: query,
         spellCheck: false,
-        ref: "input"
+        ref: c => (this.input = c)
```

Comment:
i prefer `$` if we do the property style. I still like ReactDOM.findNode or `this.ref` as well.